tasks/enable_pxc80_experimental_repo.yml

```diff
@@ -15,7 +15,7 @@
     when: ansible_os_family == "RedHat"
 
   - name: change the default repo to experimental
-    command: percona-release enable-only pxc-80 experimental
+    command: percona-release enable pxc-80 experimental
 
```

Comment:
without the `-only` part this will leave the `original` repo (with all its packages enabled). Other option is that we do `percona-release disable all` at the beginning.

Reply:
Added back `enabled-only`